src/features/components/atoms/image/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { ImgHTMLAttributes } from 'react'
+import Box from '../box'
+import './image.scss'
+import dclsx from 'features/utils/dclsx'
+
+interface ImageProps extends ImgHTMLAttributes<'img'> {
+    has_rtl?: boolean
+}
+
+const Image = ({ className, has_rtl, ...rest }: ImageProps) => {
```

Comment:
What do you say if we'll try to use this component for `GatsbyImage` and `StaticImage` as wrapper?
They can decrease size, improve quality and add some features like `loading` what will good for performance of website